krux_boto/util.py

```diff
@@ -39,3 +40,29 @@ def get_instance_region():
         get_logger('krux_boto').warn('get_instance_region failed to get the local instance region')
         raise Error('get_instance_region failed to get the local instance region')
     return zone.rstrip(string.lowercase)
+
+
+# Region codes
+class RegionCode(Enum):
+    ASH = ('ASH', 'us-east-1')
+    PDX = ('PDX', 'us-west-2')
+    DUB = ('DUB', 'eu-west-1')
+    SIN = ('SIN', 'ap-southeast-1')
```

Comment:
Instead of extending this from `Enum` we could extend from `dict` and make it "read-only" by throwing a `NotImplementedError` from the `__setitem__` method. 

It'd also remove the `for` loop on line 64.